networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -1434,4 +1435,67 @@ TEST_F(CentralityGTest, testGroupCloseness) {
 
 	EXPECT_NEAR(gc.scoreOfGroup(apx), 1.0, 1e-5);
 }
+
+TEST_F(CentralityGTest, testKadabraAbsolute) {
```

Comment:
The two tests here can fail e.g. if we change the random number generator, right? Even though it is unlikely, there could be more than `delta * n` errors.